configtools/__init__.py

```diff
@@ -5,10 +5,10 @@
 
 try:
     # python3
-    from configparser import SafeConfigParser
+    from configparser import ConfigParser
 except:
     from ConfigParser import SafeConfigParser
```

Comment:
Why even keep this option if we no longer support python2?

Reply:
Got rid of the python2 tests in .travis.yml and simplified this as you suggest.